packages/ove-service-layout/docs/LAYOUTS.md

```diff
@@ -0,0 +1,46 @@
+# Layouts
+
+**Note:** At the moment, all layout parameters are relative to the parent, 
+meaning that all their geometries will be shifted by the parent (x,y):
+
+```js
+container.x = 50
+container.section.x = 30
+// the resulting section geometry will be parent + section
+container.section.geometry.x = 80
+```
+
+## Static
+
+**Name:** static  
+**Parent parameters:** n/a  
+**Parameters:** 
+- x: displacement on the x axis, relative to the parent, **numeric**, **required**
+- y: displacement on the y axis, relative to the parent, **numeric**, **required**
+- w: width, **numeric**, **required**
+- h: height, **numeric**, **required**
+
+## Percent
+
+**Name:** percent  
+**Parent parameters:** n/a  
+**Parameters:** 
+- x: percent of the parent width for displacement on the x axis, relative to the parent, **percent**, **required**
+- y: percent of the parent height for displacement on the y axis, relative to the parent, **percent**, **required**
+- w: percent of the parent width, **percent**, **required**
+- h: percent of the parent height, **percent**, **required**
+
+**Note:** All percent parameters are between \[0, 1\]
```

Comment:
Usually in the web world, percent parameters are between 0 and 100. Why are we using a decimal notation instead?

Reply:
So I don't have to divide by 100 (which is what most of the web stuff does). It is just a convention I adopted, but we can change it if someone else finds this confusing. We are not supposed to write these layout files by hand anyway ...

Reply:
![image](https://user-images.githubusercontent.com/1260383/47152861-cc2a6e80-d2d5-11e8-8d3c-718699b64ca4.png)

Reply:
I changed everything to use the percent definition